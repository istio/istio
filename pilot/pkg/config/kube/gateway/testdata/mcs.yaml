apiVersion: gateway.networking.k8s.io/v1
kind: Gateway
metadata:
  name: gateway
  namespace: istio-system
spec:
  addresses:
  - value: istio-ingressgateway
    type: Hostname
  gatewayClassName: istio
  listeners:
  - name: default
    port: 34000
    protocol: TCP
---
apiVersion: gateway.networking.k8s.io/v1alpha2
kind: TCPRoute
metadata:
  name: tcp
  namespace: istio-system
spec:
  parentRefs:
  - name: gateway
    namespace: istio-system
  rules:
  - backendRefs:
    - group: multicluster.x-k8s.io
      kind: ServiceImport
      name: echo
      port: 80
