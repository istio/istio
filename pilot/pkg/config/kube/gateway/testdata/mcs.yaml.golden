apiVersion: networking.istio.io/v1
kind: Gateway
metadata:
  annotations:
    internal.istio.io/gateway-semantics: gateway
    internal.istio.io/gateway-service: istio-ingressgateway.istio-system.svc.domain.suffix
    internal.istio.io/parents: Gateway/gateway/default.istio-system
    internal.istio.io/service-account-name: ""
  name: gateway~istio-autogenerated-k8s-gateway~default
  namespace: istio-system
spec:
  servers:
  - hosts:
    - istio-system/*
    port:
      name: default
      number: 34000
      protocol: TCP
---
apiVersion: networking.istio.io/v1
kind: VirtualService
metadata:
  annotations:
    internal.istio.io/parents: TCPRoute/tcp.istio-system
    internal.istio.io/route-semantics: gateway
  name: tcp~tcp~0~istio-autogenerated-k8s-gateway
  namespace: istio-system
spec:
  gateways:
  - istio-system/gateway~istio-autogenerated-k8s-gateway~default
  hosts:
  - '*'
  tcp:
  - route:
    - destination:
        host: echo.istio-system.svc.domain.suffix
        port:
          number: 80
---
